### PR TITLE
Implement marketplace UI

### PR DESCRIPTION
**Pull request by lin8231**
* Marketplace UI
* Half of the page is already linked with logic (particularly the right column) 
* Left side is UI only atm
* Decision made to have search button in place so that the chart data is queried when and only when user clicks search, instead of doing live queries (with every keystroke) - this is due to each query taking ~5 seconds which would make live querying infeasible
* Factored out a Button component - which is just a simple button with styling
Loooks liket his
Screen Shot 2021-02-05 at 11 14 58 am

### CODE REVIEW

File: project/frontend/src/containers/Marketplace/Marketplace.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+const Marketplace = (props) => {
+  console.log(props);
+  return (
+    <div className={classes.MarketplaceContainer}>
+      <div className={classes.BuyContainer}>
+        <div className={classes.SearchContainer}>
+          <input
+            className={classes.SearchInput}
+            placeholder="Google search here"
```
Comment (leonnemets) [inline project/frontend/src/containers/Marketplace/Marketplace.js:16]: Feels like the word Trend should be in the placeholder

**Review by leonnemets — APPROVED** [review]
Especially nice description of the changes being made and more importantly why

File: project/frontend/src/containers/Marketplace/Marketplace.js (hunk 1 above)
Comment (lin8231) [reply project/frontend/src/containers/Marketplace/Marketplace.js]: CHange d it to "Search a trend"

**Review by roonil-wazlib — APPROVED** [review]
LGTM

**Pull request merged**